### PR TITLE
Add "Getting started with nteract" guide

### PR DESCRIPTION
**Pull request by captainsafia**

### CODE REVIEW

File: user-guides/getting-started-with-nteract.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
+# Getting started with nteract
+
+Interactive notebooks are a great tool for data scientists, hobbyists, and programmers looking to document their code, experiment with models, generate visualizations, and prototype code. The Jupyter notebook format is one of the most popular interactive notebook formats and is supported by the nteract ecosystem. This user guide will help you get set up with the basics of the nteract application.
```
Comment (rgbkrk) [inline user-guides/getting-started-with-nteract.md]: I kind of feel like the last line here might end better with `...get set up with the basics of the nteract notebook app.`

File: user-guides/getting-started-with-nteract.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+### Setting up the nteract desktop application
+
+The fastest way to get started with nteract is to download the desktop application to your machine. You can download the installer for your operating system on the [nteract homepage](https://nteract.io/desktop). Once you've downloaded and installed nteract, you can launch the program from the install directory. Note that this guide assumes you are using the nteract desktop application.
```
Comment (rgbkrk) [inline user-guides/getting-started-with-nteract.md]: The last note here seems redundant within the paragraph. If anything, I'd move this disclaimer about the guide assuming they're using desktop to the top of the page.

File: user-guides/getting-started-with-nteract.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
```
Comment (rgbkrk) [inline user-guides/getting-started-with-nteract.md]: ... and now I realize that the web section is next

I guess I should read the rest of this first.

File: user-guides/getting-started-with-nteract.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+$ pip install nteract_on_jupyter
+```
+
+Then, run Jupyter with the nteract extension installed using the following command.
+
+```bash
+$ jupyter nteract
+```
+
+Your default browser should open to the nteract web interface where you can open, create, and edit notebooks.
```
Comment (rgbkrk) [inline user-guides/getting-started-with-nteract.md]: 🎉

File: user-guides/getting-started-with-nteract.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+
+### How to create a new notebook
+
```
Comment (rgbkrk) [inline user-guides/getting-started-with-nteract.md]: Do we mention that the desktop app opens up a fresh notebook for you on launch?

File: user-guides/getting-started-with-nteract.md [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+### Organizing cells within a notebook
+
+Like any written document, the organization of the content matters. nteract allows you to rearrange the cells within a notebook using drag-and-drop. You can click on the left-hand side of a cell and drag it to the desired position.
+
+![Using drag and drop to rearrange cells in a notebook](https://cldup.com/HgCrZl9zXW.gif)
+
+### Speed things up with keyboard shortcuts
+
+As you use notebooks more and more, you'll probably want to start taking advantage of some of the keyboard shortcuts that are provided within nteract. You can view a full list of these keyboard shortcuts, and how they compare to those provided within the JupyterLab and classic Jupyter front ends [here](https://docs.nteract.io/#/desktop/shortcut-keys).
```
Comment (rgbkrk) [inline user-guides/getting-started-with-nteract.md]: I would link the `view a full list of these keyboard shortcuts` like this:

```markdown
You can view a [full list of these keyboard shortcuts](https://docs.nteract.io/#/desktop/shortcut-keys) to get yourself comfortable.
```

File: user-guides/getting-started-with-nteract.md [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+
+The nteract Data Explorer is a built-in tool for visualizing pandas DataFrames within a notebook. To see an example of the Data Explorer in action, navigate to File > Open Example Notebook > python > Try the Data Explorer. By default, the Data Explorer renders the data in a rich, paginated table.
+
+![Data rendered in a rich table within the nteract Data Explorer](https://cldup.com/-0UimYM3wH.png)
+
+Using the buttons on the right of the Data Explorer, you can switch to different views of the same data. For example, here's the data from the table categorized and filtered into a bar plot.
+
+![The nteract Data Explorer rendering a customized bar plot](https://cldup.com/54aF-NFjS3.png)
+
+At this point, you're hopefully excited about all the notebooks you can create using nteract. You can view some of the example notebooks to view the possibilities that are available to you with nteract. You can open the example notebooks by selecting File > Open Example Notebook from the nteract menubar.
```
Comment (rgbkrk) [inline user-guides/getting-started-with-nteract.md]: Probably need a heading on this section, maybe `## Good Luck!` or something silly like that.

**Comment by captainsafia** [comment]
Thanks for the feedback! I've addressed the issues.

**Pull request merged**